eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -761,6 +761,29 @@ groups:
           max:     +10.591752810763625
           latex:    '$\alpha_{T,2}^{B \to D,\mathrm{BSZ2015}}$'
           unit:     '1'
+      # form factor parameters for B->D for [BFW:2010A] parametrization
+      'B->D::a^%1%_%2%@BFW2010' :
+          central:  0.
+          min:     -1.0
+          max:     +1.0
+          unit:     '1'
+          latex:    '$a_%2%^{%1%,B \to D,\mathrm{BFW2010}}$'
+          matrix:
+            - ["f+", "f0", "fT"]
+            - [0, 1, 2, 3, 4]
+      'B->D::tp@BFW2010' :
+          central:  51.1082
+          min:      51.1082
+          max:      51.1082
+          latex:    '$t_+^{B\to D}$'
+          comment:  '(mB + mD)^2'
+          unit:     'GeV'
```

Comment:
The unit is incorrect

Reply:
I will simply remove the BFW2010 commit for the moment being